projects/igniteui-angular/src/lib/grids/headers/grid-header.component.ts

```diff
@@ -83,6 +91,13 @@ export class IgxGridHeaderComponent implements DoCheck, OnDestroy {
         return null;
     }
 
+    /**
+     * @hidden
+     */
+    public get iconTemplate() {
```

Comment:
Use more specific property name.
```suggestion
    public get esfIconTemplate() {
```